var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -12,12 +12,15 @@ class ParallelNetcdf(AutotoolsPackage):
     Unidata's NetCDF."""
 
     homepage = "https://trac.mcs.anl.gov/projects/parallel-netcdf"
-    url      = "http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/parallel-netcdf-1.6.1.tar.gz"
+    url      = "http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/parallel-netcdf-1.10.0.tar.gz"
     list_url = "http://cucis.ece.northwestern.edu/projects/PnetCDF/download.html"
 
-    version('1.8.0', '825825481aa629eb82f21ca37afff1609b8eeb07')
-    version('1.7.0', '267eab7b6f9dc78c4d0e6def2def3aea4bc7c9f0')
-    version('1.6.1', '62a094eb952f9d1e15f07d56e535052604f1ac34')
```

Comment:
Changes in hashes are potentially dangerous, because they represent an potential opportunity to slip in a doctored version.  What is Spack's policy on converting to sha256?

Reply:
I'm not sure, but it is the default that is returned when running `spack checksum {package}` so I thought it would be the best to use.  A little more secure than older md5, but maybe better to use sha385 or sha512.  I just used the current default...